lib/coralogix.js

```diff
@@ -37,6 +37,7 @@ let coralogixStats = {};
 let coralogixPrefix;
 let coralogixMappings = [];
 let coralogixConfig = {};
+let totalsAccumulator = new Map()
```

Comment:
Ideally, we should have some TTL for the values in this map, otherwise it will grow infinitely.